### PR TITLE
Enable sync cursor for multifile project when there is noly one preview server

### PR DESCRIPTION
**Pull request by 7sDream**
As discussion in #164, this is a incomplete, but simple and acceptable temporary solution for multifile project.

### CODE REVIEW

File: addons/vscode/src/extension.ts
Comment (Enter-tainer) [inline addons/vscode/src/extension.ts:100]: can you add some comment about the hack here?

File: addons/vscode/src/extension.ts
Comment (Enter-tainer) [inline addons/vscode/src/extension.ts:283]: same here

**Review by Enter-tainer — APPROVED** [review]
cool

**Comment by 7sDream** [comment]
Ccomments added.

The `Allow edits by maintainers` option of this PR is enabled, feel free to do changes you want before merge.

**Review by Enter-tainer — APPROVED** [review]
thx

**Pull request merged**
